test-suite/shared/src/test/scala/org/scalajs/testsuite/javalib/util/concurrent/ConcurrentHashMapTest.scala

```diff
@@ -194,6 +194,17 @@ class ConcurrentHashMapTest extends MapTest {
         assertTrue(s"missing value '$value'", encounteredValues.contains(value))
     }
   }
+
+  @Test def addIsUnsupportedOnKeySet(): Unit = {
+    val chm = factory.empty[String, String]
+    val set = chm.keySet()
+
+    expectThrows(classOf[UnsupportedOperationException], set.add("ONE"))
+    expectThrows(classOf[UnsupportedOperationException], set.add(null))
+    expectThrows(classOf[UnsupportedOperationException], set.addAll(null))
+    expectThrows(classOf[UnsupportedOperationException], set.addAll(ju.Arrays.asList(null)))
+    expectThrows(classOf[UnsupportedOperationException], set.addAll(ju.Arrays.asList("ONE")))
+  }
```

Comment:
This should go to `MapTest`.

Reply:
Moved to `MapTest` https://github.com/scala-js/scala-js/pull/4367/commits/d636996881b26218886a3f03952105e710b408a9